src/bokeh/models/formatters.py

```diff
@@ -653,6 +653,13 @@ def __init__(self, *args, **kwargs) -> None:
     as they are on boundary of the days scale: ["06/08", "06h", "12h", 18h", "06/09"]
     """)
 
+    hide_repeats = Bool(default=False, help="""
+    When activated only the first value of repeating formatted ticks will be shown. E.g. ["06/07", "06/07", "06/07",
+    "06/08", "06/08"] will become ["06/07", "", "", "06/08", ""]. In order to determine if a repeated tick label should
+    be hidden, only the base label without context is considered, leaving the context unaffected. Note: If the context
+    itself is a DatetimeTickFormatter the logic above can be specifically applied only to the context.
+    """)
```

Comment:
We also try to be consistent with the verbiage for boolean flags and have them start by stating that are to specify *"Whether the thing will occur"*. What do you think of the version below? 

> Whether repeated formatted tick values will be suppressed. For example, an initial set of ticks ["06/07", "06/07", "06/07",  "06/08", "06/08"] will become ["06/07", "", "", "06/08", ""]. Only the base label, without any additional context, is considered. (If the context itself is a DateTimeTickformatter, then this property may be set on that context separately).

Reply:
Sounds good to me.